packages/playground/wordpress/src/boot.ts

```diff
@@ -0,0 +1,141 @@
+import {
+	BasePHP,
+	PHPRequestHandler,
+	SpawnHandler,
+	proxyFileSystem,
+	rotatePHPRuntime,
+	writeFiles,
+} from '@php-wasm/universal';
+import {
+	enablePlatformMuPlugins,
+	preloadPhpInfoRoute,
+	preloadRequiredMuPlugin,
+} from '.';
+
+export type PhpIniOptions = Record<string, string>;
+export type Hook = () => void | Promise<void>;
+export interface Hooks {
+	beforeWordPress?: Hook;
+	beforeDatabase?: Hook;
+	beforeBlueprint?: Hook;
+}
+
+export type DatabaseType = 'sqlite' | 'mysql' | 'custom';
+
+export interface BootOptions<PHP extends BasePHP> {
+	createPhpRuntime: () => Promise<number>;
+	createPhpInstance: () => PHP;
+	/** Default: 'sqlite' */
+	databaseType?: DatabaseType;
+	/**
+	 * Mounting and Copying is handled via hooks for starters.
+	 *
+	 * In the future we could standardize the
+	 * browser-specific and node-specific mounts
+	 * in the future.
+	 */
+	hooks?: Hooks;
+	siteUrl: string;
+	/** SQL file to load instead of installing WordPress. */
+	dataSqlPath?: string;
+	/** Zip with the WordPress installation to extract in /wordpress. */
+	baseSnapshot?: File | Promise<File>;
+	/** Preloaded SQLite integration plugin. */
+	sqliteIntegrationPlugin?: File | Promise<File>;
+	spawnHandler?: SpawnHandler;
+	phpIniPath?: string;
+	phpIniOverrides?: PhpIniOptions;
+	phpIniRoute?: string;
+	muPlugins?: Record<string, string>;
+	sharedFiles?: Record<string, string>;
+}
+
+export async function bootPlayground<PHP extends BasePHP>(
+	options: BootOptions<PHP>
+) {
+	async function createPhp(
+		requestHandler: PHPRequestHandler<BasePHP>,
+		isPrimary: boolean
+	) {
+		const php = options.createPhpInstance();
+		php.initializeRuntime(await options.createPhpRuntime());
+		if (requestHandler) {
+			php.requestHandler = requestHandler;
+		}
+		if (options.phpIniPath) {
+			php.setPhpIniPath(options.phpIniPath);
+		}
+		for (const [key, value] of Object.entries(
+			options.phpIniOverrides || {}
+		)) {
+			php.setPhpIniEntry(key, value);
+		}
```

Comment:
This funciton could use some comments to make create steps easier to understand. 
We could achieve something similar by moving each step to it's own funciton. For example this code into a `setPhpIniEntrires` funciton.

Reply:
Good call, I added more comments.